app.js

```diff
@@ -1,17 +1,10 @@
 'use strict';
 var express = require('express'),
-	fs = require('fs'),
-	JSON5 = require('json5'),
-	app = express(),
-	jsonData;
+    questionsJS = require('./questions.js'),
+	app = express();
 
-var questions = [];
-
-fs.readFile('./public/freebase_countries.json', 'utf8', function(err, data){
-    jsonData = JSON5.parse(data);
-    getDataFromJson(jsonData, questions);
-    console.log(questions);
-});
+var questionsList = questionsJS.correctQuestions();
+console.log(questionsList);
```

Comment:
why do you print countries in app.js? I didn't ask you to do it.

Leave questions only in questions.js.